packages/terra-application-navigation/docs/example.md

```diff
@@ -0,0 +1 @@
+# Terra Application Navigation
```

Comment:
This read me has no content. I feel like this is for dev-site, but could we include a header in the react rendering?

Reply:
It's used to match markdown headings of the other pages using the doc template.

Reply:
I think you could just inline it in the example.jsx ```readme={'# Terra Application Navigation'}```